src/app/ubs-admin/components/ubs-admin-employee/ubs-admin-employee.component.ts

```diff
@@ -0,0 +1,52 @@
+import { UbsAdminEmployeeService } from './../../services/ubs-admin-employee.service';
+import { Location } from '@angular/common';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Params } from '@angular/router';
+import { takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+@Component({
+  selector: 'app-ubs-admin-employee',
+  templateUrl: './ubs-admin-employee.component.html',
+  styleUrls: ['./ubs-admin-employee.component.scss']
+})
+export class UbsAdminEmployeeComponent implements OnInit {
+  fakeData: string[] = [];
+  destroy: Subject<boolean> = new Subject<boolean>();
+  totalLength: number;
+  currentPage = 1;
+  paginPage: number;
+  size = 5;
+  paginationId = 'employee';
+
+  constructor(
+    private activatedRoute: ActivatedRoute,
+    private location: Location,
+    private ubsAdminEmployeeService: UbsAdminEmployeeService
+  ) {}
+
+  ngOnInit(): void {
+    this.activatedRoute.params.subscribe((params: Params) => {
+      this.paginPage = params.page - 1;
+    });
+    this.getEmployees(this.paginPage, this.size);
```

Comment:
there is no need to pass this params, because you can access them in method

Reply:
? Maybe i don't understand something, but i need to pass these params.

Reply:
it's not necessary, in getEmployees you can access this, and get the same values

Reply:
Done and works